src/loader.php

```diff
@@ -1,9 +1,13 @@
 <?php
+/**
+ * @copyright 2012-2018 Hostnet B.V.
+ */
+declare(strict_types=1);
 
 if (file_exists(__DIR__ .'/../vendor/autoload.php')) {
-    require_once __DIR__ .'/../vendor/autoload.php';
+    include_once __DIR__ .'/../vendor/autoload.php';
```

Comment:
Why is this changed?

Reply:
phpcbf must've changed this, should it be changed back?

Reply:
> require is identical to include except upon failure it will also produce a fatal E_COMPILE_ERROR level error. In other words, it will halt the script whereas include only emits a warning (E_WARNING) which allows the script to continue. 

http://php.net/manual/en/function.require.php

So yes, that should be changed back to require. Phpcbf should not be fixing this as it's not a code style but feature :thinking: